### ISSUE TITLE
Adding Contributor section in readme  ( Automatically )

### ISSUE THREAD

**Issue by rohank2502** [vinitshahdeo/PortScanner#140]
I will like to add a feature this will update contributor list in read me automatically !! Will start to work on this issue as soon as i get assigend !!

**Comment by rohank2502** [comment]
Should i go with it or not ? @vinitshahdeo

**Comment by vinitshahdeo** [comment]
@rohank2502 Yes, please go ahead. Previously we had **Hall of fame** by [sourcerer.io](https://sourcerer.io/), unfortunately it got deprecated.

**Comment by vinitshahdeo** [comment]
@rohank2502 Yes, please go ahead. Previously we had **Hall of fame** by [sourcerer.io](https://sourcerer.io/), unfortunately it got deprecated.

**Comment by rohank2502** [comment]
Please look at the Pr

**Comment by ig77ayush** [comment]
Please assign this issue to me
